clients/src/test/java/org/apache/kafka/clients/consumer/internals/HeartbeatRequestManagerTest.java

```diff
@@ -152,6 +152,34 @@ public void cleanup() {
         }
     }
 
+    @Test
+    public void testHeartBeatRequestStateToStringBase() {
+        final long retryBackoffMs = 100;
```

Comment:
i think we should try to consistently apply final to all variables if we want to use final.